ak_vendor/report/report.py

```diff
@@ -518,6 +529,10 @@ def add_owaspmobile2024(
     def add_pcidss(self, pcidss: PCIDSS) -> List[PCIDSS]:
         self.pcidss.append(pcidss)
         return self.pcidss
+    
+    def add_pcidss4(self, pcidss: PCIDSS4) -> List[PCIDSS4]:
+        self.pcidss4.append(pcidss)
+        return self.pcidss
```

Comment:
The return value will be `self.pcidss4`. Also change the method argument to `pcidss4` instead of `pcidss` to avoid any confusion,